src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  min-width: calcMaxWidth($card-width, $card-side-gap, $catalog-side-padding);
+  max-width: calcMaxWidth($card-width, $card-side-gap, $catalog-side-padding);
```

Comment:
maybe easier use just width? it doesn't make sense when you use the same values for max and min